src/main/java/com/checkmarx/flow/custom/GitHubIssueTracker.java

```diff
@@ -19,6 +18,7 @@
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
+
 import java.util.ArrayList;
```

Comment:
minor :)

Reply:
What do you mean?

Reply:
I mean that you have an extra blank space in here :)

Reply:
This is by design. Try ctrl+alt+o in intellij.